flyway/Makefile

```diff
@@ -0,0 +1,64 @@
+ARCH:=$(shell uname -m)
+EXTENSION:=$(shell case "$(ARCH)" in \
+	("arm64"|"aarch64") echo ".arm64" ;; \
+	("x86_64") echo "" ;; \
+	(*) echo $(ARCH) ;; \
+esac)
+
+.PHONY: build
+build:
+	@find . -type f -name "Dockerfile${EXTENSION}" | while read -r FILE; do \
+		docker build -f Dockerfile -t chatwork/`basename $$PWD` .; \
+      	version=$$(docker inspect -f {{.Config.Labels.version}} chatwork/`basename $$PWD`); \
+      	if [ -n "$$version" ]; then \
+            docker tag chatwork/`basename $$PWD`:latest chatwork/`basename $$PWD`:$$version; \
+        fi \
+    done
+
+.PHONY: check
+check:
+	@version=$$(docker inspect -f {{.Config.Labels.version}} chatwork/`basename $$PWD`); \
+		if [ -z "$$version" ]; then \
+			echo "\033[91mError: version is not defined in Dockerfile.\033[0m"; \
+			exit 1; \
+		fi;
+	@echo "\033[92mno problem.\033[0m";
+
+.PHONY: test
+test:
+	docker-compose -f docker-compose.test.yml up --build --no-start sut
+	docker cp $(shell pwd)/goss `basename $$PWD`:/goss
+	docker-compose -f docker-compose.test.yml up --no-recreate --exit-code-from sut sut
+
+.PHONY: push
+push:
+	@version=$$(docker inspect -f {{.Config.Labels.version}} chatwork/`basename $$PWD`:latest); \
+		if docker inspect --format='{{index .RepoDigests 0}}' chatwork/$$(basename $$PWD):$$version >/dev/null 2>&1; then \
+			echo "no changes"; \
+		else \
+		  	docker tag chatwork/`basename $$PWD`:latest chatwork/`basename $$PWD`:$$version-$(ARCH); \
+		  	docker push chatwork/`basename $$PWD`:$$version-$(ARCH); \
+		fi
+
+.PHONY: manifest\:push
+manifest\:push:
+	@version=$$(docker inspect -f {{.Config.Labels.version}} chatwork/`basename $$PWD`:latest); \
+	if docker inspect --format='{{index .RepoDigests 0}}' chatwork/$$(basename $$PWD):$$version >/dev/null 2>&1; then \
+	  	echo "no changes"; \
+	else \
+		docker pull chatwork/flyway:$$version-aarch64; \
+		docker pull chatwork/flyway:$$version-x86_64; \
```

Comment:
@tshinowpub 
I think the error occurs when I create a new version.
Is `docker pull` a necessary action to do `docker push`?

Reply:
@cw-ozaki 

docker pull is necessary. 

This flow is running on the x86_64 flow, which requires two digest values, arm64 and amd64.

Reply:
@cw-ozaki 

What kind of version tag do you envision when you say new version?